Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
Why not, my addition of isIdentifier to AttributeMetadata was my last dev but it's more simple to use. 

But if I move the logic of IdentifierLoader and DoctrineIdentifierLoader into AttributesLoader why having AnnotationLoader for example ?

Or maybe move IdentifierLoader and DoctrineIdentifierLoader  into AnnotationLoader, because it's annotations. But @sroze expose a point : AttributeMetadata are constructed into AttributeLoader and if we want to secure isIdentifier ...

For the multiple primary keys, actually my DoctrineLoader ignore the annotation if it's a multiple primary key. And it's the second loader at the end, wich check if there are one identifier.
These issue require to parse all attributes, before setting the identifier to an attribute so ... 

IMO : 
The best way here is to move all the logic into AnnotationLoader : 
* First this loader parse all attributes and determinate wich have DoctrineId, our own Identifier annotation, and Id attribute.
* Second : we determine the real identifier by priorities : 
 * own identifier set first (error if we have two annotation of this kind during parsing)
 * doctrine id if there is a simple key 
 * id properties if exist and public (default)
* Third : we set into the correct attribute the "isIdentifier" property

Warning : these dev require to parse all attributes in AttributesLoader (on master we ignore the identifier).  Another problem appears : the identifier can appear alone in the serialization. (I start another thread for this question)